content/en/blog/2024/introducing-hydrophone.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: blog
+title: "Introducing Hydrophone"
+slug: introducing-hydrophone-2024
```

Comment:
(nit)
```suggestion
slug: introducing-hydrophone
```

(We're unlikely to launch a second Hydrophone subproject, so no need for a year here)